src/openms/source/FORMAT/IdXMLFile.cpp

```diff
@@ -101,13 +101,13 @@ namespace OpenMS
 
     //write header
     os << "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n";
-    os << "<?xml-stylesheet type=\"text/xsl\" href=\"http://open-ms.sourceforge.net/XSL/IdXML.xsl\" ?>\n";
+    os << "<?xml-stylesheet type=\"text/xsl\" href=\"https://raw.githubusercontent.com/OpenMS/OpenMS/develop/share/OpenMS/XSL/IdXML.xsl\" ?>\n";
```

Comment:
well, the schema should be stable, regardless of the branch. Has to start 'living' somewhere outside sourceforge ... - If you want to open a release 2.1 branch, we can put it there.